ShestakUI/Config/Filters/FilgerSpells.lua

```diff
@@ -892,6 +892,8 @@ C["filger_spells"] = {
 			IconSize = C.filger.buffs_size,
 			Position = {"TOP", P_BUFF_ICON_Anchor},
 
+			-- Aspect of the Turtle
+			{spellID = 186265, unitID = "player", caster = "player", filter = "BUFF"},
```

Comment:
Ah crap! I didn't notice that, I was gonna add it to the PVP section so I thought it must have been missing from the Hunter section too! I'll re-do the pull request!